infra/worker/workers-set.yaml

```diff
@@ -28,10 +29,23 @@ spec:
           value: firestore://projects/ossf-malware-analysis/databases/(default)/documents/
         securityContext:
           privileged: true
+        volumeMounts:
+          - mountPath: "/var/lib/containers"
+            name: image-storage
```

Comment:
Did you have any thoughts on how we can automate the invalidation of this cache overtime?

Reply:
We always pull the latest sandbox image here: https://github.com/ossf/package-analysis/blob/75b7264f7262505c6889e23b1a1f3e277812410b/internal/sandbox/sandbox.go#L92 so we should have the latest image.

But yeah, I think this does leave around stale layers over time. We may need to call `podman image prune` or somewhere.

Reply:
Added a call to that in `internal/sandbox/sandbox.go`

Reply:
nice!